tests/test_spatialstats.py

```diff
@@ -434,9 +433,9 @@ def test_sample_empirical_variogram_speed(self) -> None:
         subsample = 10
 
         # First, run the xDEM wrapper function
-        t0 = time.time()
```

Comment:
Weird that this shows up, since it's already fixed in the merged #355. Or am I missing something?

Reply:
if it's new, perhaps refer to the issue to make it easier to backtrack what changed what in the future

Reply:
One that was forgotten, refering back.